csunplugged/topics/models.py

```diff
@@ -45,6 +45,7 @@ class CurriculumArea(models.Model):
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(unique=True)
     name = models.CharField(max_length=100, unique=True)
+    colour = models.CharField(max_length=15, null=True)
```

Comment:
This is because it's only a CSS class name, 15 characters seemed generous enough for a class name but can expand it if you think it would be a problem.